homeassistant/components/advantage_air/climate.py

```diff
@@ -0,0 +1,287 @@
+"""Climate platform for Advantage Air integration."""
+import logging
+
+from homeassistant.components.climate import ClimateEntity
+from homeassistant.components.climate.const import (
+    FAN_AUTO,
+    FAN_HIGH,
+    FAN_LOW,
+    FAN_MEDIUM,
+    FAN_OFF,
+    HVAC_MODE_COOL,
+    HVAC_MODE_DRY,
+    HVAC_MODE_FAN_ONLY,
+    HVAC_MODE_HEAT,
+    HVAC_MODE_OFF,
+    SUPPORT_FAN_MODE,
+    SUPPORT_TARGET_TEMPERATURE,
+)
+from homeassistant.const import (
+    ATTR_TEMPERATURE,
+    CONF_IP_ADDRESS,
+    PRECISION_WHOLE,
+    STATE_OFF,
+    STATE_ON,
+    STATE_OPEN,
+    TEMP_CELSIUS,
+)
+from homeassistant.helpers.update_coordinator import CoordinatorEntity
+
+from .const import DOMAIN, STATE_CLOSE
+
+ADVANTAGE_AIR_HVAC_MODES = {
+    "heat": HVAC_MODE_HEAT,
+    "cool": HVAC_MODE_COOL,
+    "vent": HVAC_MODE_FAN_ONLY,
+    "dry": HVAC_MODE_DRY,
+}
+HASS_HVAC_MODES = {v: k for k, v in ADVANTAGE_AIR_HVAC_MODES.items()}
+
+ADVANTAGE_AIR_FAN_MODES = {
+    "auto": FAN_AUTO,
+    "low": FAN_LOW,
+    "medium": FAN_MEDIUM,
+    "high": FAN_HIGH,
+}
+HASS_FAN_MODES = {v: k for k, v in ADVANTAGE_AIR_FAN_MODES.items()}
+FAN_SPEEDS = {FAN_LOW: 30, FAN_MEDIUM: 60, FAN_HIGH: 100}
+
+_LOGGER = logging.getLogger(__name__)
+
+
+async def async_setup_platform(hass, config, async_add_entities, discovery_info=None):
+    """Platform setup isn't required."""
+    return True
+
+
+async def async_setup_entry(hass, config_entry, async_add_entities):
+    """Set up AdvantageAir climate platform."""
+
+    instance = hass.data[DOMAIN][config_entry.data[CONF_IP_ADDRESS]]
+
+    entities = []
+    for _, ac_index in enumerate(instance["coordinator"].data["aircons"]):
```

Comment:
isn't the return value of enumerate (index, value)?

Reply:
Yes, I throw the index out using underscore. I believe I am using this to loop over the dictionary of {"ac1":{},"ac2":{}}

Reply:
I have removed enumerate()